### DISCUSSION TITLE
Welcome to Mod20_Segment_2 Discussions!

### DISCUSSION THREAD

**Discussion by maryamt95** [ALIYA2Group/Mod20_Segment_2#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by maryamt95** [comment]
we got the data , and this is the north data preview .
![image](https://user-images.githubusercontent.com/89698299/149874061-65e3b459-42e1-419a-af24-791295e8a72f.png)

**Reply by maryamt95** [reply]
with renamed titles 

![image](https://user-images.githubusercontent.com/89698299/149874252-ec6cdaf6-277b-49f8-92fa-cb62cd2a105a.png)

**Comment by 735713038455163** [comment]
![image](https://user-images.githubusercontent.com/88687569/150661136-74a9ff6e-69d2-4500-9f78-380bae1334e5.png)

This was uploaded to R
